source/particles/generators.inst.in

```diff
@@ -51,6 +51,34 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
             &particle_handler,
           const Mapping<deal_II_dimension, deal_II_space_dimension> &mapping,
           const unsigned int random_number_seed);
+
+#  if deal_II_dimension > 1
```

Comment:
It might not be particularly exciting, but why not instantiate this for 1d as well?

Reply:
One of the function called therein (I don't remember exactly which) is not instantiated for 1D, so the code actually does not compile for 1D because of this. That is the only reason. I will look-up which function is not instantiated for 1D exactly.

Reply:
I see. If you can isolate what function that is, then we can do this here as a follow-up.